packages/blockchain-extension/extension/commands/associateTransactionDataDirectoryCommand.ts

```diff
@@ -65,7 +67,18 @@ export async function associateTransactionDataDirectory(chaincode?: Instantiated
 
     gateway = await FabricGatewayConnectionManager.instance().getGatewayRegistryEntry();
 
-    const quickPickItems: string[] = [UserInputUtil.BROWSE_LABEL];
+    const quickPickItems: Array<{ label: string, description: string }> = [];
```

Comment:
That should be of type `IBlockchainQuickPickItem` then you can put the path as the data even if you also have the path as the description too